README.md

```diff
@@ -28,5 +28,7 @@ pip install kfserving
 ```
 * Follow the [example here](docs/samples/client/kfserving_sdk_sample.ipynb) to use the KFServing SDK to create, patch, and delete a InferenceService instance.
 
+Note: If you are running kfserving sdk on a machine and want to point to a remote kfserving cluster, please initialize it with a pointer to kube config file location, e.g. `KFServingClient(config_file=/path/to/kubeconfig)`. If not provided, it defaults to `~/.kube/config`.
```

Comment:
I don't think this note belongs in the main README. Maybe docs/samples/client/kfserving_sdk_sample.ipynb?

Reply:
Ok - will do a follow-on. This was a bigger stumbling block for me - and hence thought surfacing it right at the outset

Reply:
@ellis-bigelow @animeshsingh I already add some description in the #437 for this. I think we should add one link for SDK document in tghe main README as doing in #437 since someone asked me where can found the SDK docs. I will remove the added section in the #437  :-)